CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to Transmission
+
+:+1::tada: Thanks for reading this and considering contributing to the project! :tada::+1:
+
+The following is a set of guidelines for contributing to Transmission. These are just guidelines, not rules, use your best judgment and feel free to propose changes to this document in a pull request.
+
+# What changes would be welcome?
+
+- Bugfixes
+- Changes that improve Transmission's compliance with [accepted BEPs](https://www.bittorrent.org/beps/bep_0000.html)
+- Changes that improve transfer speeds
+- Changes that improve peer communication, indirectly improving transfer speeds
+- Changes that measurably reduce CPU load
+- Changes that measurably reduce memory use
+- Changes that improve testing
+- Changes that simplify or shrink the existing codebase
+- Changes that implement an issue that has a [`pr welcome` label](https://github.com/transmission/transmission/issues?q=is%3Aissue+is%3Aopen+label%3A%22pr+welcome%22)
```

Comment:
Might as well add `help wanted` and `good first issue` here, too

Reply:
Added in e74de16815c512c90d7a994fbe3d1b258c33dd8b